### DISCUSSION TITLE
Data columns in the data file

### DISCUSSION THREAD

**Discussion by sarafarahabadi** [sovelluskontti/project-semantic-search#25] [General]
Hello Turo, 

I have a question about the data file regarding its columns. I have made the manticore full-text search work with only ID and TITLE present in the DB table. I want to add more columns to check if the search still works. If it is needed for the project. 
In the data file, there are several columns:
- primaryTitle ( which I just put as "title")
- originalTitle
- isAdult
- startYear
- endYear
- runtimeMinutes
- genres

For the final prototype, which of the columns are necessary to be in the search function? 
Is the search only for the title or other columns are also needed for searching? 

Best regards, 
Sara

**Comment by nyluntu** [comment]
Hi.

For searching the PrimaryTitle is only needed. All the other fields are not need to be searchable (full text field). For exercise you should define other data types for the other fields ([attributes fields](https://manual.manticoresearch.com/Creating_a_table/Data_types#Attributes)). I think the content of originalTitle is not important so that column may be left out.

We can use them later for sorting and filtering results for example. One feature that needs attributes is [Faceted search](https://manual.manticoresearch.com/Searching/Faceted_search).

**Comment by sarafarahabadi** [comment]
Hello, 

Thanks. I will continue working on the semantic search since full-text search works now on "title". Later I will focus on this feature for sorting and filtering.
